src/cmd-create-legacy-oscontainer

```diff
@@ -126,6 +134,14 @@ with open(digestfile) as f:
     osc_digest = f.read().strip()
 
 # Inject the oscontainer with SHA256 into the build metadata
+sha256 = hashlib.sha256()
+with open(f"{latest_build_path}/{args.name}", "rb") as f:
+    for b in iter(lambda: f.read(4096), b""):
+        sha256.update(b)
+size_bytes = os.path.getsize(f"{latest_build_path}/{args.name}")
+meta['images']['legacy-oscontainer'] = {'path': args.name,
+                                        'sha256': sha256.hexdigest(),
+                                        'size': size_bytes}
 meta['oscontainer'] = {'image': args.name,
                        'digest': osc_digest}
```

Comment:
IIUC we should no longer be doing this as we aren't actually pushing a container here.

Reply:
If I remove it the upgrade fails, 

> Is the update to go modules (vendoring) required in this PR? Can we split it to a separate PR?

I was under the impression that any change to the schema would need that. If not then I can not do it in this PR or not at all.

Reply:
> If I remove it the upgrade fails,

You mean the upgrade test fails in CI?

Reply:
yes

Reply:
@dustymabe see: https://github.com/coreos/coreos-assembler/pull/3111#issuecomment-1272331105